Doc/whatsnew/3.14.rst

```diff
@@ -175,6 +175,10 @@ Improved Error Messages
 Other Language Changes
 ======================
 
+* The :func:`map` function now has an optional ``strict`` flag like
```

Comment:
[Please see the devguide](https://devguide.python.org/documentation/markup/#quick-reference) for how to properly mark up parameters and arguments.

```suggestion
* The :func:`map` function now has an optional *strict* flag like
```

Reply:
This sentence was copied from here: https://docs.python.org/3/whatsnew/3.10.html#other-language-changes
Do I change it there too?

Reply:
No